projects/js-packages/publicize-components/src/components/one-click-sharing-dropdown/index.js

```diff
@@ -0,0 +1,69 @@
+import { Text, ThemeProvider } from '@automattic/jetpack-components';
+import { Button, Dropdown } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { Icon } from '@wordpress/icons';
+import { useCallback } from 'react';
+import { ShareButtons } from '../share-buttons/share-buttons';
+import shareArrowIcon from './share-arrow-icon';
+import styles from './styles.module.scss';
+
+const renderToggle = ( { isOpen, onToggle } ) => {
+	return (
+		<Icon
+			className={ styles.icon }
+			size={ 18 }
+			icon={ shareArrowIcon }
+			onClick={ onToggle }
+			aria-expanded={ isOpen }
+		/>
+	);
+};
+
+const OneClickSharingDropdown = ( { className: containerClass, onClickLearnMore } ) => {
+	const renderContent = useCallback(
+		( { onToggle } ) => {
+			return (
+				<div className="one-click-sharing-dropdown">
+					<ThemeProvider>
+						<ShareButtons buttonStyle="icon-text" />
+						<hr className={ styles.divider } />
+						<Text className={ styles.description } variant="body-small">
+							{ __( 'Share with a Single Click!', 'jetpack' ) +
+								'✨ ' +
```

Comment:
I'm not sure about this. I know it's decoration, but we're breaking the text up. Could we add a placeholder for it (like we would with a link or component) and a translator note? I'm a bit concerned that it wouldn't make sense to be placed here in certain languages

Reply:
It is not part of the translated string. So I think it's better this way.

Reply:
It's not part of the translated string because we've broken it up. This has a unicode character with a trailing space, which make is like a piece of punctuation. I feel like it would be better to have one translated string with a placeholder for the problematic character, which we could replace and also provide a translator note, so that they know what it is.